api/src/test/java/org/apache/commons/rdf/api/AbstractRDFTest.java

```diff
@@ -194,6 +194,114 @@ public void testCreateLiteralLangISO693_3() throws Exception {
         assertEquals("\"Herbert Van de Sompel\"@vls", vls.ntriplesString());
     }
 
+    public void testCreateLiteralLangCaseInsensitive() throws Exception {
+        // COMMONSRDF-51: Literal langtag may not be in lowercase, but
+        // must be COMPARED (aka .equals and .hashCode()) in lowercase
+        // as the language space is lower case.       
+        final Literal lower = factory.createLiteral("Hello", "en-gb"); 
+        final Literal upper = factory.createLiteral("Hello", "EN-GB"); 
+        final Literal mixed = factory.createLiteral("Hello", "en-GB");
+
+        
+        assertEquals("en-gb", lower.getLanguageTag().get());
```

Comment:
RDF4J may not follow this in some cases. It may use the BCP47 normalisation conventions to obtain en-GB instead.